app/components/institution-page/campus-program-application-modal.ts

```diff
@@ -16,6 +18,8 @@ interface Signature {
 
 export default class CampusProgramApplicationModal extends Component<Signature> {
   @service declare authenticator: AuthenticatorService;
+  @tracked emailAddressToPrefill: string | undefined = undefined;
+  @tracked forceShowEnterEmail = false;
```

Comment:
Doesn't really adhere to our naming conventions for boolean fields like this. Would search for all other usages of `@tracked ... = false` and see what names we use: 

<img width="881" height="786" alt="Screenshot 2025-08-16 at 02 38 35" src="https://github.com/user-attachments/assets/d6ff9235-15fa-4d2b-9a0f-7ed5020a696c" />

And let's find something similar to those names. Good rules of thumb: it is of the form `isDoingSomething` / `somethingWasDone` / `somethingIsXYZ` etc.